mipengine/node_tasks_DTOs.py

```diff
@@ -69,24 +69,44 @@ class TableInfo(ImmutableBaseModel):
     _validate_identifier = validator("name", allow_reuse=True)(validate_identifier)
 
 
-class TableView(ImmutableBaseModel):
-    datasets: List[str]
-    columns: List[str]
-    filter: Dict
+class _ColumnData(BaseModel):
```

Comment:
why redefining the same class instead of one implementation and just moving it in a common place that will be used by all modules needing this?